<launch>

<!-- mcptam_client node -->
<node name="mcptam_client" pkg="mcptam" type="mcptam_client" clear_params="true" output="screen">
  <rosparam command="load" file="$(find mcptam)/groups/$(arg group_name).yaml" />
  <!--
  <remap from="/mcptam_client/cam_group_list" to="/camera_manager/cam_group_list" />
  -->
  
  <remap from="modify_map" to="mcptam_server/modify_map"/>
  <param name="image_transport" type="string" value="compressed"/> 

  <param name="get_pose_separately" type="bool" value="false" />
  <param name="dynamic_sync" type="bool" value="true" />
  
  <!--  <param name="camera_pose_file" type="string" value="$(find mcptam)/poses/poses.dat" />  -->

  <!--
  <rosparam command="load" file="$(find mcptam)/masks/masks_1,2.yaml" />
  <param name="masks_dir" type="string" value="$(find mcptam)/masks" />
  -->

  <param name="kf_distance_mean_diff_fraction" type="double" value="0.5" />
  <param name="mm_max_scaled_mkf_dist" type="double" value="0.3" />
  <param name="mm_outlier_multiplier" type="double" value="1.1" />
  
  <!-- For no GUI, set headless:=true -->
  <param name="headless" type="bool" value="$(arg headless)" />

</node>
 
<node pkg="tf" type="static_transform_publisher" name="world_tf" args="0 0 0 -0.5 0.5 -0.5 0.5 world vision_world 100" />
 
</launch>

